diff --git a/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/RNGestureHandlerPackage.kt b/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/RNGestureHandlerPackage.kt
index e271a3e..79c66f0 100644
--- a/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/RNGestureHandlerPackage.kt
+++ b/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/RNGestureHandlerPackage.kt
@@ -36,13 +36,17 @@ class RNGestureHandlerPackage :
   override fun createViewManagers(reactContext: ReactApplicationContext) =
     listOf<ViewManager<*, *>>(RNGestureHandlerRootViewManager(), RNGestureHandlerButtonViewManager())
 
-  override fun getViewManagerNames(reactContext: ReactApplicationContext) = viewManagers.keys.toList()
+  override fun getViewManagerNames(reactContext: ReactApplicationContext): Collection<String> = 
+    viewManagers.keys
 
   override fun getViewManagers(reactContext: ReactApplicationContext): MutableList<ModuleSpec> =
     viewManagers.values.toMutableList()
 
-  override fun createViewManager(reactContext: ReactApplicationContext, viewManagerName: String) =
-    viewManagers[viewManagerName]?.provider?.get() as? ViewManager<*, *>
+  override fun createViewManager(
+    reactContext: ReactApplicationContext,
+    viewManagerName: String,
+  ): ViewManager<in Nothing, in Nothing>? =
+    viewManagers[viewManagerName]?.provider?.get() as? ViewManager<in Nothing, in Nothing>
 
   override fun getModule(name: String, reactContext: ReactApplicationContext): NativeModule? =
     if (name == RNGestureHandlerModule.NAME) {
